examples/README.md

```diff
@@ -0,0 +1 @@
+This folder contains working examples of usage of the library.
```

Comment:
I'd like to make a wiki, once the graphql_ppx and your more configurable graphql client has been merged

Reply:
You mean instead of the example? The main reason to have them for me is to have working compiled code in Reason, it helps also while developing the modules/bindings.